intel-sgx/pcs/src/tcb_info.rs

```diff
@@ -135,8 +136,8 @@ fn sgx_platform() -> Platform {
 pub struct TcbData<V: VerificationType = Verified> {
     id: Platform,
     version: u16,
-    issue_date: String,
-    next_update: String,
+    issue_date: DateTime<Utc>,
+    next_update: DateTime<Utc>,
```

Comment:
There is a foot gun here, if we add `derive(Serialize)` for this in the future and forget to annotate the fields with `#[serde(with = "crate::iso8601")]`. Why not add a wrapper type for serialization/deserialization logic?

Reply:
Adding `derive(Serialize)` here may be an issue in itself as we wouldn't be able to verify the struct anymore when the serialized data doesn't match exactly with how Intel created it. The `TcbInfo` already derives serialize and avoids that issue by storing the `raw_tcb_info` as a `String`

Reply:
Let me merge the PR now. We can sort this out in another PR if we reach another conclusion.